### DISCUSSION TITLE
限定共有記事で編集リクエストを受け付けたい

### DISCUSSION THREAD

**Discussion by lvn-tanaka** [increments/qiita-discussions#575] [要望・アイデア・不具合]
記事を全体に公開する前に、チームメンバーなどに記事のレビューをお願いすることがあります。
そのときに、編集リクエスト機能が利用できると、レビューがはかどります。
現状、限定共有記事で編集リクエスト機能が利用できないのには、何か理由があるのでしょうか。差し支えなければ、限定共有記事においても、編集リクエスト機能を利用できるようにしていただけると助かります。

**Comment by tanaka-asami415** [comment]
@cs-tanaka 
リクエストありがとうございます🙏

限定共有記事に編集リクエスト機能をご提供できていない理由の１つとして
編集リクエストを送ることで Contribution数が加算されるため、
全体公開記事との区別をどのようにするか慎重に検討する必要があることがあります。

限定共有記事への編集リクエスト機能を検討したことはあるのですが、上記理由により一旦見送りとなっている状況です。

直近の開発優先度などからすぐの対応はできませんが、引き続き検討はしたいと考えておりますので
今回いただいたご意見も参考にさせていただきます！

また、第三者のレビューを経てご投稿いただく方法として、下記もご案内させていただきますので
よろしければご検討ください🙇

### Qiita CLIをご活用いただく https://github.com/increments/qiita-cli
Qiita CLIをご活用いただくと記事を GitHub上で管理できますので、
GitHub の機能により第三者によるレビューを経て投稿していただくことが可能です。

### Qiita Teamをご活用いただく https://teams.qiita.com/
Qiita Teamはチーム間でのクローズドな情報共有ができるサービスです（有料）
下記機能をご活用いただいた上で、Qiitaにご投稿いただく手法もございます。
- チームメンバーと共同で記事を作成できる「共同編集モード」
- Qiita Teamの記事からQiitaの下書きを作成する機能

**Reply by lvn-tanaka** [reply]
ご回答ありがとうございます。
ぜひ、ご検討をお願いします。

限定共有記事への編集リクエストについては Contribution数は加算しない仕様がよいかと思います。
